pages/index.js

```diff
@@ -1,19 +1,82 @@
 import Button from "../components/Button";
 import React, { useState } from "react";
 
+// Zustände für ausgewählte Option, ausgewähltes Datum, eingereichte Daten und Anzeige der Eingabemöglichkeiten
 export default function HomePage() {
-  function handleClick() {}
+  const [selectedOption, setSelectedOption] = useState("");
+  const [selectedDate, setSelectedDate] = useState("");
+  const [submittedData, setSubmittedData] = useState([]);
+  const [showInput, setShowInput] = useState(false);
+
+  // Funktion zum Verarbeiten der eingegebenen Daten
+  const handleSubmit = () => {
+    if (selectedOption && selectedDate) {
+      const newData = { option: selectedOption, date: selectedDate };
+      setSubmittedData([...submittedData, newData]);
+      setSelectedOption("");
+      setSelectedDate("");
+    }
```

Comment:
Once you wrapped a form around your input fields and extracted the event, you can receive the event in this function and create two const here in this function. These variables (const) can be local. they don't need to be states on their own, as all the information is already saved within submittedData anyway 
```suggestion
function handleSubmit (event) {
event.preventDefault()
const formElements = event.target.elements;
const selectedOption = formElements.option.value;
const selectedDate = formElements.date.value;

..(continue logic to create object for each task)..

}
```